packages/math-input/src/components/keypad/index.tsx

```diff
@@ -82,8 +84,28 @@ export default function Keypad(props: Props) {
         basicRelations,
         advancedRelations,
         showDismiss,
+        sendEvent,
     } = props;
 
+    useEffect(() => {
+        if (!isMounted) {
+            sendEvent({
+                type: "perseus:keypad-opened",
```

Comment:
I was thinking that these events would be formatted as `package:event`... so given this originates from `math-input`, I'd format it as `math-input:keypad-opened`.  Sorry, I didn't document this anywhere yet as analytics in the Perseus repo are I flux.